app/models/plate.rb

```diff
@@ -210,53 +221,14 @@ def iteration
     iter.zero? ? nil : iter # Maintains compatibility with legacy version
   end
 
-  class CommentsProxy
```

Comment:
THis has been moved. And a comment has been added to explain why this monstrosity exists.